src/Command/RunCommand.php

```diff
@@ -300,4 +251,22 @@ private function writeReports(ReportBuffer $report, ProjectConfiguration $projec
             }
         }
     }
+
+    private function getCores(): int
+    {
+        if ('/' === DIRECTORY_SEPARATOR) {
+            $process = new Process(['nproc']);
+            try {
+                $process->mustRun();
+                return (int) trim($process->getOutput());
+            } catch (Throwable $ignored) {
+                // fallback.
+                $process = new Process(['grep', '-c', '^processor', '/proc/cpuinfo']);
+                $process->mustRun();
```

Comment:
This should be catched as well and `1` should be returned in case of an error.

Reply:
Addressed in c5a098b